extra/3-magic-8-ball.js

```diff
@@ -58,61 +134,79 @@ function shakeBall() {
 
   This function should expect to be called with any value which was returned by the shakeBall function.
 */
-function checkAnswer(answer) {
-  //Write your code in here
+
+function checkAnswer(magicAnswer) {
+	//Write your code in here
+	var index = answers.indexOf(magicAnswer);
+	var status;
+	if (index < 5) {
+		status = "very positive";
+	} else if (index >= 5 && index < 10) {
+		status = "positive";
+	} else if (index >= 10 && index < 15) {
+		status = "negative";
+	} else if (index >= 15 && index < 20) {
+		status = "very negative";
+	}
```

Comment:
Do you need 2 parts to your if statements in lines 144, 146? Have you already established one of them? Do you need the check on line 148 at all?